courses-app/src/services/services.js

```diff
@@ -0,0 +1,27 @@
+export default class Service {
+	_apiBase = 'http://localhost:3000';
+
+	async userLogin(body) {
+		const response = await fetch(this._apiBase + '/login', {
+			method: 'POST',
+			body: JSON.stringify(body),
+			headers: {
+				'Content-Type': 'application/json',
+			},
+		});

```

Comment:
add util somewhere to get or post (accumulate configuring of the requests in one place)

Reply:
ty, done